cmd/genautocomplete/genautocomplete_test.go

```diff
@@ -21,6 +21,23 @@ func TestCompletionBash(t *testing.T) {
 	assert.NotEmpty(t, string(bs))
 }
 
+func TestCompletionBashStdout(t *testing.T) {
+	originalStdout := os.Stdout
+	readOut, writeOut, errorPipe := os.Pipe()
+	assert.NoError(t, errorPipe)
+
+	os.Stdout = writeOut
+	bashCommandDefinition.Run(bashCommandDefinition, []string{"-"})
+
+	_ = writeOut.Close()
+	output, err := ioutil.ReadAll(readOut)
+
+	assert.NoError(t, err)
+	assert.NotEmpty(t, string(output))
+
+	os.Stdout = originalStdout
```

Comment:
This should be in a `defer() { }()` just below `os.Stdout = writeOut`

Reply:
An empty `defer`? What does this accomplish?

Reply:
I feel so stupid sometimes. These 3 selected line or just the assignment?